_includes/pbc/all/install-features/202505.0/install-the-order-amendment-feature.md

```diff
@@ -1527,6 +1608,80 @@ class PriceProductSalesOrderAmendmentDependencyProvider extends SprykerPriceProd
 
 
 
+{% endinfo_block %}
+
+## Add quotation process context
```

Comment:
Shouldn't be the other QR related plugins be added in this section but not in the main one?

Reply:
make sense